bundles/org.openhab.binding.salus/src/main/java/org/openhab/binding/salus/internal/rest/exceptions/SalusApiException.java

```diff
@@ -10,14 +10,17 @@
  *
  * SPDX-License-Identifier: EPL-2.0
  */
-package org.openhab.binding.salus.internal.rest;
+package org.openhab.binding.salus.internal.rest.exceptions;
 
 import java.io.Serial;
 
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
 /**
  * @author Martin Grześlowski - Initial contribution
  */
 @SuppressWarnings("SerializableHasSerializationMethods")
```

Comment:
See other exception, can you generate a `serialVersionUID` and remove the suppression? (missed that in the previous PR)

Reply:
https://github.com/openhab/openhab-addons/pull/16807#discussion_r1616365084

Reply:
done